_data/simple-icons.json

```
@@ -615,6 +615,11 @@
             "hex": "0066DC",
             "source": "http://press.dailymotion.com/?page_id=346"
         },
+        {
+            "title": "Dassault Systemes",
```

I'm sorry for not being clear, this title should stay as it was 😅 We want this value to accurately reflect the brand name, whereas the filename of the SVG is a simplified version without spaces and non-English characters.

```suggestion
            "title": "Dassault Systèmes",
```